src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -1010,6 +1010,36 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.Windows.targets" Condition="'$(TargetPlatformIdentifier)' == 'Windows'" />
   <Import Project="$(MSBuildThisFileDirectory)../../Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.targets" Condition="'$(ImportWindowsDesktopTargets)' == 'true'"/>
 
+  <!-- Workload support -->
+
+  <PropertyGroup>
+    <!-- TODO: This should be based on NETCoreSdkVersion, and probably set with it in BundledVersions.props-->
+    <SdkFeatureBand>5.0.100</SdkFeatureBand>
+
+    <!-- Default to environment variable -->
+    <WorkloadManifestRoot Condition="'$(WorkloadManifestRoot)' == ''">$(DOTNETSDK_WORKLOAD_MANIFEST_ROOT)</WorkloadManifestRoot>
+
+    <!-- TODO: Remove this (it's for local testing) -->
+    <!--<WorkloadManifestRoot Condition="'$(WorkloadManifestRoot)' == ''">C:\git\msbuild-sdk-resolver-test\testresolver\sdk-manifests\$(SdkFeatureBand)</WorkloadManifestRoot>-->
+   
+    <!-- TODO: This should probably go in BundledVersions.props-->
+    <WorkloadManifestRoot Condition="'$(WorkloadManifestRoot)' == ''">$([MSBuild]::EnsureTrailingSlash('$(NetCoreRoot)'))sdk-manifests\$(SdkFeatureBand)</WorkloadManifestRoot>
```

Comment:
SdkFeatureBand and the ManifestRoot is calcuated in manifest reader too. We should only keep one logic

Reply:
This is addressed now by using a new pseudo-SDK (Microsoft.NET.SDK.WorkloadManifestTargetsLocator) so the resolver can return the right paths.